sdk/digitaltwins/azure-digitaltwins-core/src/main/java/com/azure/digitaltwins/core/models/DeleteDigitalTwinRequestOptions.java

```diff
@@ -1,25 +0,0 @@
-// Copyright (c) Microsoft Corporation. All rights reserved.
-// Licensed under the MIT License.
-
-package com.azure.digitaltwins.core.models;
-
-import com.azure.digitaltwins.core.DigitalTwinsClient;
```

Comment:
We get this type generated for free, so no need for this manually defined type anymore